pkg/cloud/services/compute/instance.go

```diff
@@ -221,6 +229,10 @@ func createInstance(is *Service, clusterName string, i *infrav1.Instance) (*infr
 			}
 		}
 	}
+	if i.AccessSubnet != "" && accessIPv4 == "" {
+		return nil, fmt.Errorf("no ports with fixed IPs found on AccessSubnet \"%s\"", i.AccessSubnet)
```

Comment:
If AccessSubnet is renamed to Subnet, This comment would be updated.

Reply:
:heavy_check_mark: